nautobot/extras/models/approvals.py

```diff
@@ -261,7 +261,7 @@ def save(self, *args, **kwargs):
 
         if previous_state != self.current_state:
             if self.current_state == ApprovalWorkflowStateChoices.APPROVED:
-                self.object_under_review.on_workflow_approved()
+                self.object_under_review.on_workflow_approved(self)
             elif self.current_state == ApprovalWorkflowStateChoices.DENIED:
                 self.object_under_review.on_workflow_denied()
```

Comment:
For consistency, probably should change `on_workflow_denied` (and maybe `on_workflow_initiated` too?) signature as well?